core/src/main/java/org/mapfish/print/map/style/StyleParser.java

```diff
@@ -49,10 +49,8 @@
                             userLayer.addUserStyle(style.get());
                             final StyledLayerDescriptor sld = styleFactory.createStyledLayerDescriptor();
                             sld.addStyledLayer(userLayer);
-                            if (LOGGER.isDebugEnabled()) {
-                                LOGGER.debug(String.format("Loaded style from: \n\n '%s': \n\n%s",
-                                        styleString, transformer.transform(sld)));
-                            }
+                            LOGGER.debug("Loaded style from: \n\n '{}': \n\n{}",
+                                    styleString, transformer.transform(sld));
```

Comment:
I guess transformer.transform is quite costly. I wouldn't remove the `if` in that case...